+++
title = "Running in circles: is practical application feasible for data fission and data thinning in post-clustering differential analysis?"
date = "2024-05-26"

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["B Hivert", "D Agniel", "R Thi√©baut", "<u>**BP Hejblum**</u>"]
# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference proceedings
# 2 = Journal
# 3 = Work in progress
# 4 = Technical report
# 5 = Book
# 6 = Book chapter
# 7 = Preprint
publication_types = ["7"]

# Publication name and optional abbreviated version.
publication = "*arXiv* 2405.13591"
publication_short = "*arXiv* 2405.13591"

doi ="10.48550/arXiv.2405.13591"

# Abstract and optional shortened version.
abstract = "The standard pipeline to analyse single-cell RNA sequencing (scRNA-seq) often involves two steps : clustering and Differential Expression Analysis (DEA) to annotate cell populations based on gene expression. However, using clustering results for data-driven hypothesis formulation compromises statistical properties, especially Type I error control. Data fission was introduced to split the information contained in each observation into two independent parts that can be used for clustering and testing. However, data fission was originally designed for non-mixture distributions, and adapting it for mixtures requires knowledge of the unknown clustering structure to estimate component-specific scale parameters. As components are typically unavailable in practice, scale parameter estimators often exhibit bias. We explicitly quantify how this bias affects subsequent post-clustering differential analysis Type I error rate despite employing data fission. In response, we propose a novel approach that involves modeling each observation as a realization of its distribution, with scale parameters estimated non-parametrically. Simulations study showcase the efficacy of our method when component are clearly separated. However, the level of separability required to reach good performance presents complexities in its application to real scRNA-seq data."
abstract_short = ""

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = true

# Links (optional).
url_pdf = "https://arxiv.org/pdf/2405.13591"
url_preprint = "https://doi.org/10.48550/arXiv.2405.13591"
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
# Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
url_custom = [{name = "R code", url = "https://doi.org/10.5281/zenodo.1120777"}]


# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = false

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

+++
