---
title: Scientific publications
slug: publications
---



<p><a href="/publication"><strong><em>&gt; Interactive publication list here</em></strong></a></p>
<p>*: equal contribution</p>
<div id="section" class="section level1">
<h1><strong>2025</strong></h1>
<ul>
<li><p>Ferté T, Ba K, Dutartre D, Legrand P, Jouhet V, Thiébaut R, Hinaut X &amp; <u><strong>Hejblum BP</strong></u>. Reservoir computing in r: A tutorial for using reservoirnet to predict complex time-series. <em>Computo</em>(in press):, 2025.</p></li>
<li><p>Chouleur T*, Etchegaray C*, Villain L*, Lesur A, Ferté T, Rossi M, Andrique L, Simoncini C, Giacobbi A, Gambaretti M, Lopci E, Fernades B, Dittmar G, Bjerkvig R, <u><strong>Hejblum B</strong></u>, Thiébaut R, Saut O, Bello L &amp; Bikfalvi A. A strategy for multimodal integration of transcriptomics, proteomics, and radiomics data for the prediction of recurrence in patients with IDH-mutant gliomas. <em>International Journal of Cancer</em>, 2025.<br />
DOI: <a href="https://doi.org/10.1002/ijc.35441">10.1002/ijc.35441</a></p></li>
<li><p>Zhou D, Tong H, Wang L, Liu S, Xiong X, Gan Z, Griffier R, <u><strong>Hejblum B</strong></u>, Liu Y, Hong C, Bonzel C, Cai T, Pan K, Ho Y, Costa L, Panickan VA, Gaziano JM, Mandl K, Jouhet V, Thiebaut R, Xia Z, Cho K, Liao K &amp; Cai T. Representation learning to advance multi-institutional studies with electronic health record data. <em>arXiv</em> 2502.08547, 2025.<br />
DOI: <a href="https://doi.org/10.48550/arXiv.2502.08547">10.48550/arXiv.2502.08547</a></p></li>
<li><p>Hughes A, Parast L, Thiébaut R &amp; <u><strong>Hejblum BP</strong></u>. Rank-based identification of high-dimensional surrogate markers: Application to vaccinology. <em>arXiv</em> 2502.03030, 2025.<br />
DOI: <a href="https://doi.org/10.48550/arXiv.2502.03030">10.48550/arXiv.2502.03030</a></p></li>
</ul>
</div>
<div id="section-1" class="section level1">
<h1><strong>2024</strong></h1>
<ul>
<li><p>Hivert B, Agniel D, Thiébaut R &amp; <u><strong>Hejblum BP</strong></u>. Post-clustering difference testing: Valid inference and practical considerations with applications to ecological and biological data. <em>Computational Statistics &amp; Data Analysis</em> 107916, 2024.<br />
DOI: <a href="https://doi.org/10.1016/j.csda.2023.107916">10.1016/j.csda.2023.107916</a></p></li>
<li><p><u><strong>Hejblum BP</strong></u>, Ba K, Thiébaut R &amp; Agniel D. Neglecting normalization impact in semi-synthetic RNA-Seq data simulation generates artificial false positives. <em>Genome Biology</em> 25:281, 2024.<br />
DOI: <a href="https://doi.org/10.1186/s13059-024-03231-9">10.1186/s13059-024-03231-9</a></p></li>
<li><p>Ferté T, Dutartre D, <u><strong>Hejblum BP</strong></u>, Griffier R, Jouhet V, Thiébaut R, Legrand P &amp; Hinaut X. Reservoir computing for short high-dimensional time series: An application to SARS-CoV-2 hospitalization forecast. <em>Proceedings of the 41st international conference on machine learning</em> 235:13570-13591, 2024.</p></li>
<li><p>Collin A, <u><strong>Hejblum BP</strong></u>, Vignals C, Lehot L, Thiébaut R, Moireau P &amp; Prague M. Using population based kalman estimator to model COVID-19 epidemic in france: Estimating the effects of non-pharmaceutical interventions on the dynamics of epidemic. <em>International Journal of Biostatistics</em> 20(1):13-41, 2024.<br />
DOI: <a href="https://doi.org/10.1515/ijb-2022-0087">10.1515/ijb-2022-0087</a></p></li>
<li><p>Hivert B, Agniel D, Thiébaut R &amp; <u><strong>Hejblum BP</strong></u>. Running in circles: Practical limitations for real-life application of data fission and data thinning in post-clustering differential analysis. <em>arXiv</em> 2405.13591, 2024.<br />
DOI: <a href="https://doi.org/10.48550/arXiv.2405.13591">10.48550/arXiv.2405.13591</a></p></li>
<li><p>White E, Papagno L, Samri A, Sugata K, <u><strong>Hejblum B</strong></u>, Henry AR, Rogan DC, Darko S, Recordon-Pinson P, Dudoit Y, Llewellyn-Lacey LA, Buseyne F, Migueles SA, Price DA, Andreola M, Satou Y, Thiébaut R, Katlama C, Autran B, C DD &amp; Appay V. Clonal succession after prolonged antiretroviral therapy rejuvenates CD8+ t cell responses against HIV-1. <em>Nature Immunology</em> 25:1555-1564, 2024.<br />
DOI: <a href="https://doi.org/10.1038/s41590-024-01931-9">10.1038/s41590-024-01931-9</a></p></li>
</ul>
</div>
<div id="section-2" class="section level1">
<h1><strong>2023</strong></h1>
<ul>
<li><p>Freulon P, Bigot J &amp; <u><strong>Hejblum BP</strong></u>. CytOpT: Optimal Transport with Domain Adaptation for Interpreting Flow Cytometry data. <em>Annals of Applied Statistics</em> 17(2):1086-1104, 2023.<br />
DOI: <a href="https://doi.org/10.1214/22-AOAS1660">10.1214/22-AOAS1660</a></p></li>
<li><p>Agniel D, <u><strong>Hejblum BP</strong></u>, Thiébaut R &amp; Parast L. Doubly-robust evaluation of high-dimensional surrogate markers. <em>Biostatistics</em> 24(4):985-999, 2023.<br />
DOI: <a href="https://doi.org/10.1093/biostatistics/kxac020">10.1093/biostatistics/kxac020</a></p></li>
<li><p>Blengio F, Hocini H, Richert L, Lefebvre C, Durand M, <u><strong>Hejblum B</strong></u>, Tisserand P, McLean C, Luhn K, Thiebaut R &amp; Lévy Y. Identification of early gene expression profiles associated with long-lasting antibody responses to the Ebola vaccine Ad26. ZEBOV/MVA-BN-Filo. <em>Cell Reports</em> 42(9):113101, 2023.<br />
DOI: <a href="https://doi.org/10.1016/j.celrep.2023.113101">10.1016/j.celrep.2023.113101</a></p></li>
<li><p>Vignals C, <u><strong>Hejblum BP</strong></u> &amp; Prague M. Modéliser la COVID-19: De la population à l’individu. <em>Interstices</em>, 2023.</p></li>
<li><p>Thiébaut R, <u><strong>Hejblum B</strong></u>, Mougin F, Tzourio C &amp; Richert L. ChatGPT and beyond with artificial intelligence (AI) in health: Lessons to be learned. <em>Joint Bone Spine</em> 90(5):105607, 2023.<br />
DOI: <a href="https://doi.org/10.1016/j.jbspin.2023.105607">10.1016/j.jbspin.2023.105607</a></p></li>
</ul>
</div>
<div id="section-3" class="section level1">
<h1><strong>2022</strong></h1>
<ul>
<li><p>Bigot J, Freulon P, <u><strong>Hejblum BP</strong></u> &amp; Leclaire A. On the potential benefits of entropic regularization for smoothing wasserstein estimators. <em>arXiv</em> 2210.06934, 2022.<br />
DOI: <a href="https://doi.org/10.48550/arXiv.2210.06934">10.48550/arXiv.2210.06934</a></p></li>
<li><p>Ferté T, Jouhet V, Greffier R, <u><strong>Hejblum BP</strong></u> &amp; Thiébaut R. The benefit of augmenting open data with clinical data-warehouse EHR for forecasting SARS-CoV-2 hospitalizations in Bordeaux area, France. <em>JAMIA open</em> ooac086, 2022.<br />
DOI: <a href="https://doi.org/10.1093/jamiaopen/ooac086">10.1093/jamiaopen/ooac086</a></p></li>
<li><p>Richert L, Lelièvre J, Lacabaratz C, Hardel L, Hocini H, Wiedemann A, Lucht F, Poizot-Martin I, Bauduin C, Diallo A, Rieux V, Durand M, <u><strong>Hejblum BP</strong></u>, Launay O, Thiébaut R, Lévy Y, on behalf of the ANRS VRI01 Study group. T-cell immunogenicity, gene expression profile and safety of four heterologous prime-boost combinations of HIV vaccine candidates in healthy volunteers - results of the randomized multi-arm phase i/II ANRS VRI01 trial. <em>Journal of Immunology</em> 208(12):2663-2674, 2022.<br />
DOI: <a href="https://doi.org/10.4049/jimmunol.2101076">10.4049/jimmunol.2101076</a></p></li>
<li><p>Rinchai D, Deola S, Zoppoli G, Ahamed Kabeer BS, Taleb S, Pavlovski I, Maacha S, Gentilcore G, Toufiq M, Mathew L, Liu L, Vempalli FR, Mubarak G, Lorenz S, Sivieri I, Cirmena G, Dentone C, Cuccarolo P, Giacobbe D, Baldi F, Garbarino A, Cigolini B, Cremonesi P, Bedognetti M, Ballestrero A, Bassetti M, <u><strong>Hejblum BP</strong></u>, Augustine T, Van Panhuys N, Thiébaut R, Branco R, Chew T, Shojaei M, Short K, Feng C, PREDICT-19 consortium, Zughaier SM, De Maria A, Tang B, Ait Hssain A, Bedognetti D, Grivel J, Chaussabel D. High–temporal resolution profiling reveals distinct immune trajectories following the first and second doses of COVID-19 mRNA vaccines. <em>Science Advances</em> 8(45):eabp9961, 2022.<br />
DOI: <a href="https://doi.org/10.1126/sciadv.abp9961">10.1126/sciadv.abp9961</a></p></li>
</ul>
</div>
<div id="section-4" class="section level1">
<h1><strong>2021</strong></h1>
<ul>
<li><p>Gauthier M, Agniel D, Thiébaut R &amp; <u><strong>Hejblum BP</strong></u>. Distribution-free complex hypothesis testing for single-cell RNA-seq differential expression analysis. <em>bioRxiv</em> 2021.05.21.445165, 2021.<br />
DOI: <a href="https://doi.org/10.1101/2021.05.21.445165">10.1101/2021.05.21.445165</a></p></li>
<li><p>Acar N, Merle BMJ, Ajana S, He Z, Grégoire S, <u><strong>Hejblum BP</strong></u>, Martine L, Buaud B, Bron AM, Creuzot-Garcher CP, Korobelnik J, Berdeaux O, Jacqmin-Gadda H, Bretillon L, Delcourt C, for the Biomarkers of Lipid Status And metabolism in Retinal ageing (BLISAR) Study Group. Predicting the retinal content in omega-3 fatty acids for age-related macular-degeneration. <em>Clinical and Translational Medicine</em> 11(7):e404, 2021.<br />
DOI: <a href="https://doi.org/10.1002/ctm2.404">10.1002/ctm2.404</a></p></li>
<li><p>Lefèvre-Arbogast S, <u><strong>Hejblum BP</strong></u>, Helmer C, Klose C, Manach C, Low DY, Urpi-Sarda M, Andres-Lacueva C, González-Domínguez R, Aigner L, Altendorfer B, Lucassen PJ, Ruigrok SR, De Lucia C, Du Preez A, Proust-Lima C, Thuret S, Korosi A &amp; Samieri C. Early signature in the blood lipidome associated with subsequent cognitive decline in the elderly: A case-control analysis nested within the three-city cohort study. <em>EBioMedicine</em> 64:103216, 2021.<br />
DOI: <a href="https://doi.org/10.1016/j.ebiom.2021.103216">10.1016/j.ebiom.2021.103216</a></p></li>
<li><p>Lévy Y, Wiedemann A*, <u><strong>Hejblum BP</strong></u>*, Durand M, Lefebvre C, Surénaud M, Lacabaratz C, Perreau M, Foucat E, Déchenaud M, Tisserand P, Blengio F, Hivert B, Gauthier M, Cervantes-Gonzalez M, Bachelet D, Laouénan C, Bouadma L, Timsit J, Yazdanpanah Y, Pantaleo G, Hocini H &amp; Thiébaut R. CD177, a specific marker of neutrophil activation, is associated with coronavirus disease 2019 severity and death. <em>iScience</em> 24(7):102711, 2021.<br />
DOI: <a href="https://doi.org/10.1016/j.isci.2021.102711">10.1016/j.isci.2021.102711</a></p></li>
<li><p>Colas C, <u><strong>Hejblum B</strong></u>, Rouillon S, Thiébaut R, Oudeyer P, Moulin-Frier C &amp; Prague M. EpidemiOptim: A toolbox for the optimization of control policies in epidemiological models. <em>Journal of Artificial Intelligence Research</em> 71:479-519, 2021.<br />
DOI: <a href="https://doi.org/10.1613/jair.1.12588">10.1613/jair.1.12588</a></p></li>
<li><p>Zhang HG*, <u><strong>Hejblum BP</strong></u>*, Weber G, Palmer N, Churchill S, Szolovits P, Murphy S, Liao K, Kohane I &amp; Cai T. ATLAS: An automated association test using probabilistically linked health records with application to genetic studies. <em>Journal of the American Medical Informatics Association</em> 28(12):2582-2592, 2021.<br />
DOI: <a href="https://doi.org/10.1093/jamia/ocab187">10.1093/jamia/ocab187</a></p></li>
<li><p>Ferté T, Cossin S, Schaeverbeke T, Barnetche T, Jouhet V &amp; <u><strong>Hejblum BP</strong></u>. Automatic phenotyping of electronical health record: PheVis algorithm. <em>Journal of Biomedical Informatics</em> 117:103746, 2021.<br />
DOI: <a href="https://doi.org/10.1016/j.jbi.2021.103746">10.1016/j.jbi.2021.103746</a></p></li>
<li><p>Ajana S, Cougnard-Grégoire A, Colijn J, Merle BM, Verzijden T, Jong P, Hofman A, EYE-RISK Consortium, Vingerling J, <u><strong>Hejblum BP</strong></u>, Korobelnik J, Meester-Smoor M, Jacqmin-Gadda H, Klaver C, Delcourt C. Predicting progression to advanced age-related macular degeneration from clinical, genetic and lifestyle factors using machine learning. <em>Ophthalmology</em> 128(4):587-597, 2021.<br />
DOI: <a href="https://doi.org/10.1016/j.ophtha.2020.08.031">10.1016/j.ophtha.2020.08.031</a></p></li>
<li><p>Philipps V, <u><strong>Hejblum BP</strong></u>, Prague M, Commenges D &amp; Proust-Lima C. Robust and efficient optimization using a marquardt- levenberg algorithm with r package marqLevAlg. <em>The R Journal</em> 13(2):365-379, 2021.<br />
DOI: <a href="https://doi.org/10.32614/RJ-2021-089">10.32614/RJ-2021-089</a></p></li>
<li><p>Villain L, Ferté T, Thiébaut R &amp; <u><strong>Hejblum BP</strong></u>. Gene set analysis for time-to-event outcome with the generalized berk-jones statistic. <em>bioRxiv</em> 2021.09.07.459329, 2021.<br />
DOI: <a href="https://doi.org/10.1101/2021.09.07.459329">10.1101/2021.09.07.459329</a></p></li>
<li><p>Lin L &amp; <u><strong>Hejblum BP</strong></u>. Bayesian mixture models for cytometry data analysis. <em>Wiley Interdisciplinary Reviews: Computational Statistics</em> 13:e1535, 2021.<br />
DOI: <a href="https://doi.org/10.1002/wics.1535">10.1002/wics.1535</a></p></li>
</ul>
</div>
<div id="section-5" class="section level1">
<h1><strong>2020</strong></h1>
<ul>
<li><p><u><strong>Hejblum BP</strong></u>, Kunzmann K, Lavagnini E, Hutchinson A, Robertson DS, Jones SC &amp; Eckes-Shephard AH. Realistic and robust reproducible research for biostatistics. <em>Preprints</em> 2020060002, 2020.<br />
DOI: <a href="https://doi.org/10.20944/preprints202006.0002.v1">10.20944/preprints202006.0002.v1</a></p></li>
<li><p>Bouadma L, Wiedemann A, Patrier J, Surenaud M, Wicky P, Foucat E, Diehl J, <u><strong>Hejblum BP</strong></u>, Sinnah F, Montmollin E, Lacabaratz C, Thiébaut R, Timsit J &amp; Lévy Y. Immune alterations during SARS-CoV-2-related acute respiratory distress syndrome. <em>Journal of Clinical Immunology</em> 40:1082-1092, 2020.<br />
DOI: <a href="https://doi.org/10.1007/s10875-020-00839-x">10.1007/s10875-020-00839-x</a></p></li>
<li><p>Lhomme E, <u><strong>Hejblum BP</strong></u>, Lacabaratz C, Wiedemann A, Lelièvre J, Lévy Y, Thiébaut R &amp; Richert L. Analyzing cellular immunogenicity in vaccine clinical trials: A new statistical method including non-specific responses for accurate estimation of vaccine effect. <em>Journal of Immunological Methods</em> 477:112711, 2020.<br />
DOI: <a href="https://doi.org/10.1016/j.jim.2019.112711">10.1016/j.jim.2019.112711</a></p></li>
<li><p>Gauthier M, Agniel D, Thiébaut R &amp; <u><strong>Hejblum BP</strong></u>. Dearseq: A variance component score test for RNA-seq differential analysis that effectively controls the false discovery rate. <em>NAR Genomics and Bioinformatics</em> 2(4):lqaa093, 2020.<br />
DOI: <a href="https://doi.org/10.1093/nargab/lqaa093">10.1093/nargab/lqaa093</a></p></li>
<li><p>Wiedemann A, Foucat E, Hocini H, Lefebvre C, <u><strong>Hejblum BP</strong></u>, Durand M, Krüger M, Keita AK, Ayouba A, Mély S, Fernandez J, Touré A, Fourati S, Lévy-Marchal C, Raoul H, Delaporte E, Koivogui L, Thiébaut R, Lacabaratz C, Lévy Y, PostEboGui Study Group. Long-lasting severe immune dysfunction in ebola virus disease survivors. <em>Nature Communications</em> 11:3730, 2020.<br />
DOI: <a href="https://doi.org/10.1038/s41467-020-17489-7">10.1038/s41467-020-17489-7</a></p></li>
<li><p>Chan SF, <u><strong>Hejblum BP</strong></u>, Chakrabortty A &amp; Cai T. Semi-supervised estimation of covariance with application to phenome-wide association studies with electronic medical records data. <em>Statistical Methods in Medical Research</em> 29:455-465, 2020.<br />
DOI: <a href="https://doi.org/10.1177/0962280219837676">10.1177/0962280219837676</a></p></li>
</ul>
</div>
<div id="section-6" class="section level1">
<h1><strong>2019</strong></h1>
<ul>
<li><p><u><strong>Hejblum BP</strong></u>, Alkhassim C, Gottardo R, Caron F &amp; Thiébaut R. Sequential dirichlet process mixture of skew t-distributions for model-based clustering of flow cytometry data. <em>Annals of Applied Statistics</em> 13(1):638-660, 2019.<br />
DOI: <a href="https://doi.org/10.1214/18-AOAS1209">10.1214/18-AOAS1209</a></p></li>
<li><p>Ajana S, Niyazi A, Bretillon L, <u><strong>Hejblum BP</strong></u>, Jacqmin-Gadda H &amp; Cécile D. Benefits of dimension reduction in penalized regression methods for high dimensional grouped data: A case study in low sample size. <em>Bioinformatics</em> 35:3628-3634, 2019.<br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btz135">10.1093/bioinformatics/btz135</a></p></li>
<li><p>Thiébaut R, <u><strong>Hejblum BP</strong></u>, Hocini H, Bonnabau H, Skinner J, Montes M, Lacabaratz C, Richert L, Palucka K, Banchereau J &amp; Levy Y. Gene expression signatures associated with immune and virological responses to therapeutic vaccination with dendritic cells in HIV-infected individuals. <em>Frontiers in Immunology</em> 10:874, 2019.<br />
DOI: <a href="https://doi.org/10.3389/fimmu.2019.00874">10.3389/fimmu.2019.00874</a></p></li>
<li><p>Low DY, Lefèvre‐Arbogast G, Urpi‐Sarda M, Micheau P, Petera M, Centeno D, Durand S, Estelle P, Korosi A, Lucassen PJ, Aigner L, Proust‐Lima C, <u><strong>Hejblum BP</strong></u>, Helmer C, Andres‐Lacueva C, Thuret S, Samieri C &amp; Manach C. Diet‐related metabolites associated with cognitive decline revealed by untargeted metabolomics in a prospective cohort. <em>Molecular Nutrition &amp; Food Research</em> 63:1900177, 2019.<br />
DOI: <a href="https://doi.org/10.1002/mnfr.201900177">10.1002/mnfr.201900177</a></p></li>
<li><p>Desquilbet L, Granger S, <u><strong>Hejblum BP</strong></u>, Legrand A, Pernot P &amp; Rougier N. Vers une recherche reproductible : Faire évoluer ses pratiques. <em>NA</em>, 2019.</p></li>
</ul>
</div>
<div id="section-7" class="section level1">
<h1><strong>2018</strong></h1>
<ul>
<li><p><u><strong>Hejblum BP</strong></u>, Cui J, Lahey LJ, Cagan A, Sparks JA, Sokolove J, Cai T &amp; Liao KP. Association between anti-citrullinated fibrinogen antibodies and coronary artery disease in rheumatoid arthritis. <em>Arthritis Care &amp; Research</em> 70:1113-1117, 2018.<br />
DOI: <a href="https://doi.org/10.1002/acr.23444">10.1002/acr.23444</a></p></li>
<li><p>Commenges D, Alkhassim C, Gottardo R, <u><strong>Hejblum BP</strong></u> &amp; Thiébaut R. Cytometree: A binary tree algorithm for automatic gating in cytometry analysis. <em>Cytometry: Part A</em> 93(11):1132-1140, 2018.<br />
DOI: <a href="https://doi.org/10.1002/cyto.a.23601">10.1002/cyto.a.23601</a></p></li>
<li><p>Sinnott JA, Cai F, Yu S, <u><strong>Hejblum BP</strong></u>, Hong C, Kohane IS &amp; Liao KP. PheProb: probabilistic phenotyping using diagnosis codes to improve power for genetic association studies. <em>Journal of the American Medical Informatics Association</em> 25(10):1359-1365, 2018.<br />
DOI: <a href="https://doi.org/10.1093/jamia/ocy056">10.1093/jamia/ocy056</a></p></li>
<li><p>Lefèvre-Arbogast S, Gaudout D, Bensalem J, Letenneur L, Dartigues J, <u><strong>Hejblum BP</strong></u>, Féart C, Delcourt C &amp; Samieri C. Pattern of polyphenol intake and the long-term risk of dementia in older persons. <em>Neurology</em>, 2018.<br />
DOI: <a href="https://doi.org/10.1212/WNL.0000000000005607">10.1212/WNL.0000000000005607</a></p></li>
<li><p><u><strong>Hejblum BP</strong></u>, Weber GM, Liao KP, Palmer NP, Churchill S, Shadick NA, Szolovits P, Murphy SN, Kohane IS &amp; Cai T. Probabilistic record linkage of de-identified research datasets with discrepancies using diagnosis codes. <em>Scientific Data</em> 6:180298, 2018.<br />
DOI: <a href="https://doi.org/10.1038/sdata.2018.298">10.1038/sdata.2018.298</a></p></li>
<li><p>Neykov M, <u><strong>Hejblum BP</strong></u> &amp; Sinnott JA. Kernel machine score test for pathway analysis in the presence of semi-competing risks. <em>Statistical Methods in Medical Research</em> 27(4):1099-1114, 2018.<br />
DOI: <a href="https://doi.org/10.1177/0962280216653427">10.1177/0962280216653427</a></p></li>
</ul>
</div>
<div id="section-8" class="section level1">
<h1><strong>2017</strong></h1>
<ul>
<li><p>Agniel D &amp; <u><strong>Hejblum BP</strong></u>. Variance component score test for time-course gene set analysis of longitudinal RNA-seq data. <em>Biostatistics</em> 18(4):589-604, 2017.<br />
DOI: <a href="https://doi.org/10.1093/biostatistics/kxx005">10.1093/biostatistics/kxx005</a></p></li>
<li><p>Rechtien A, Richert L, Lorenzo H, Martrus G, <u><strong>Hejblum B</strong></u>, Dahlke C, Kasonta R, Zinser M, Stubbe H, Matschl U, Lohse A, Krähling V, Eickmann M, Becker S, Agnandji ST, Krishna S, Kremsner PG, Brosnahan JS, Bejon P, Njuguna P, Addo MM, Becker S, Krähling V, Siegrist CA, Huttner A, Kieny MP, Moorthy V, Fast P, Savarese B, Lapujade O, Thiébaut R, Altfeld M &amp; Addo M. Systems Vaccinology Identifies an Early Innate Immune Signature as a Correlate of Antibody Responses to the Ebola Vaccine rVSV-ZEBOV. <em>Cell Reports</em> 20(9):2251-2261, 2017.<br />
DOI: <a href="https://doi.org/10.1016/j.celrep.2017.08.023">10.1016/j.celrep.2017.08.023</a></p></li>
<li><p>Liao KP*, Sparks JA*, <u><strong>Hejblum BP</strong></u>, Kuo I, Cui J, Lahey LJ, Cagan A, Gainer VS, Liu W, Cai TT, Sokolove J &amp; Cai T. Phenome‐wide association study of autoantibodies to citrullinated and noncitrullinated epitopes in rheumatoid arthritis. <em>Arthritis &amp; Rheumatology</em> 69(4):742-749, 2017.<br />
DOI: <a href="https://doi.org/10.1002/art.39974">10.1002/art.39974</a></p></li>
</ul>
</div>
<div id="section-9" class="section level1">
<h1><strong>2016</strong></h1>
<ul>
<li>Liquet B, De Micheaux PL, <u><strong>Hejblum BP</strong></u> &amp; Thiébaut R. Group and sparse group partial least square approaches applied in genomics context. <em>Bioinformatics</em> 32(1):35-42, 2016.<br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btv535">10.1093/bioinformatics/btv535</a></li>
</ul>
</div>
<div id="section-10" class="section level1">
<h1><strong>2015</strong></h1>
<ul>
<li><u><strong>Hejblum BP</strong></u>, Skinner J &amp; Thiébaut R. Time-Course Gene Set Analysis for Longitudinal Gene Expression Data. <em>PLOS Computational Biology</em> 11(6):e1004310, 2015.<br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1004310">10.1371/journal.pcbi.1004310</a></li>
</ul>
</div>
<div id="section-11" class="section level1">
<h1><strong>2014</strong></h1>
<ul>
<li><p>Furman D*, <u><strong>Hejblum BP</strong></u>*, Simon N, Jojic V, Dekker CL, Thiébaut R, Tibshirani RJ &amp; Davis MM. Systems analysis of sex differences reveals an immunosuppressive role for testosterone in the response to influenza vaccination. <em>Proceedings of the National Academy of Sciences</em> 111(2):869-874, 2014.<br />
DOI: <a href="https://doi.org/10.1073/pnas.1321060111">10.1073/pnas.1321060111</a></p></li>
<li><p>Thiébaut R, <u><strong>Hejblum BP</strong></u> &amp; Richert L. The analysis of “Big Data” in clinical research. <em>Revue d’Épidemiologie et de Santé Publique</em> 62(1):1-4, 2014.<br />
DOI: <a href="https://doi.org/10.1016/j.respe.2013.12.021">10.1016/j.respe.2013.12.021</a></p></li>
</ul>
</div>
<div id="section-12" class="section level1">
<h1><strong>2013</strong></h1>
<ul>
<li>Commenges D &amp; <u><strong>Hejblum BP</strong></u>. Evidence synthesis through a degradation model applied to myocardial infarction. <em>Lifetime Data Analysis</em> 19(1):1-18, 2013.<br />
DOI: <a href="https://doi.org/10.1007/s10985-012-9227-3">10.1007/s10985-012-9227-3</a></li>
</ul>
</div>
